partiql-plan/src/main/kotlin/org/partiql/plan/v1/rex/RexSelect.kt

```diff
@@ -11,6 +12,10 @@ public interface RexSelect : Rex {
 
     public fun getConstructor(): Rex
 
+    public override fun getType(): PType = PType.typeBag(getConstructor().getType())
```

Comment:
I'm sure you're planning on fixing everything soon -- but pointing out ordering here